server/odc-service/src/main/resources/config-meta/user-config-meta.yml

```diff
@@ -170,4 +170,14 @@
   allowed_values: [ 'true','false' ]
   default_value: 'true'
   nullable: false
-  description: 'Whether the query is being rewritten in ob-oracle mode and increases the RowId.'
\ No newline at end of file
+  description: 'Whether the query is being rewritten in ob-oracle mode and increases the RowId.'
+- id: 18
+  category: 'user'
+  key: 'odc.database.default.enableGlobalObjectSearch'
```

Comment:
The key is not the same as the system config key, is it expected?

Reply:
This is user config, it is different from system config. It is excepted.